src/scrapers/base-isracard-amex.ts

```diff
@@ -283,6 +283,17 @@ class IsracardAmexBaseScraper extends BaseScraperWithBrowser {
   }
 
   async login(credentials: ScraperCredentials): Promise<ScaperScrapingResult> {
+    await this.page.setRequestInterception(true);
+
+    this.page.on('request', (request) => {
+      // This request is blocking the page load intentionally.
+      if (request.url().endsWith('detector-dom.min.js')) {
```

Comment:
`endsWith` relies on them not to add junk at the end, consider using `includes` instead 
```suggestion
      if (request.url().includes('detector-dom.min.js')) {
```